### ISSUE TITLE
Manufacturer links will no longer be rendered as <a> starting with version > 6.7.5.0.

### ISSUE THREAD

**Issue by looserben** [shopware/platform#14055]
### Shopware Version

6.7.5.1

### Affected area / extension

Platform(Default)

### Actual behaviour

Starting with version 6.7.5.0, the manufacturer's logo on the product details page is no longer displayed as a clickable link (`<a> tag`), even though a manufacturer link is present. Instead, a `<div>` is rendered.

Probable cause: A manufacturer object is manually created in the template.

```
{% set manufacturer = {
  'media': element.data.manufacturer.media,
  'name': element.data.manufacturer.translated.name,
  'alt': element.data.manufacturer.translated.name,
  'link': config.url.value ? config.url.value : element.data.manufacturer.translated.link,
} %}
```

Later, however, a check is performed on a non-existent field:

```
{% if manufacturer.translated.link %}
  <a href="{{ manufacturer.translated.link }}">
```

The `manufacturer` object does not contain a `translated` sub-object, but only `manufacturer.link`. Therefore, the condition is always false and the `<a>` tag is not rendered.

### Expected behaviour

If a manufacturer link exists (from config.url or manufacturer.translated.link), the manufacturer logo should be rendered as a clickable <a> tag.

### How to reproduce

- Install Shopware version 6.7.5.0 or higher
- Create a product with an assigned manufacturer
- Configure the manufacturer link or CMS element URL
- Access the product details page
- Check the HTML output → no <a> tag

**Comment by Swift42** [comment]
I ran into the same problem.
Until it is officially fixed, you can circumvent the problem by creating your own cms-element-manufacturer-logo.html.twig file and use:
```
{% sw_extends '@Storefront/storefront/element/cms-element-manufacturer-logo.html.twig' %}
{% block element_manufacturer_logo_link %}
  {% set manufacturer = manufacturer|merge({'translated': { 'link': manufacturer.link } } ) %}
  {{ parent() }}
{% endblock %}
```
